<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllPathsForm</class>
 <widget class="QWidget" name="AllPathsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кратчайшие растояния в графе</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="1" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="infDistanceLabel_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Растояния более 2 147 483 647 отмечены символом ∞.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="timeLabel_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>205</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>205</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Время роботы алгоритма заняло менее </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="timeLCD_3">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timeLabel_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>73</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>68</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string> милисекунд.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QTableWidget" name="predecessorTableWidget"/>
   </item>
   <item row="1" column="1">
    <widget class="QTableWidget" name="shortestDistanceTableWidget"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="predecessorLabel">
     <property name="text">
      <string>Матрица предыдущих вершин в графе:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="shortestDistanceLabel">
     <property name="text">
      <string>Матрица кратчайших расстояний в графе:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
